docs/content/dagster-cloud/managing-deployments/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,298 @@
+│     ├── assets
+│   ├── quickstart_etl_tests
+├── dagster_cloud.yaml
+├── pyproject.toml
+├── setup.cfg
+└── setup.py
+```
+Example:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    working_directory: ./project_directory
+```
+
+### Build directory
+
+Setting a build directory is useful if your `setup.py` or `requirements.txt` is in a subdirectory instead of the project root. This is common if you have multiple Python packages within a single Dagster project.
+
+The build registry is applicable to hybrid deployments only.
+
+Example:
+
+```yaml
+# dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    build:
+      directory: ./
+      registry: your-docker-image-registry/image-name
+```
+
+### Python executable
+
+If you are using Dagster Cloud Serverless, the Python executable to use is specified [here](https://docs.dagster.io/dagster-cloud/deployment/serverless#using-a-different-python-version). For Dagster Cloud Hybrid, the python executable that is installed globally in the image, or the default Python executable on the local system if you use the local agent, will be used. If you want to use a different python executable, you can specify it in the dagster_cloud.yaml file. It might also be useful to have different python executables for different code locations.
+
+```yaml
+# dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    executable_path: venvs/path/to/dataengineering_spark_team/bin/python
+  - location_name: machine_learning
+    code_source:
+      python_file: ml_model.py
+    executable_path: venvs/path/to/ml_tensorflow/bin/python
+```
+
+---
+
+### Container context
+
+If using Hybrid deployment, you can define additional configuration options for code locations using 
+the `container_context` parameter. Depending on the Hybrid agent you're using, the configuration settings under `container_context` will vary.
+
+Refer to the configuration reference for your agent for more info: 
+
+- [Docker agent configuration reference](/dagster-cloud/deployment/agents/docker/configuration-reference)
+- [Amazon ECS agent configuration reference](/dagster-cloud/deployment/agents/amazon-ecs/configuration-reference#amazon-ecs-agent-configuration-reference)
+- [Kubernetes agent configuration reference](/dagster-cloud/deployment/agents/kubernetes/configuration-reference)
+
+
+---
+
+## Settings
+
+<ReferenceTable>
+  <ReferenceTableItem propertyName="location_name">
+    The name of your code location that will appear in the Dagster UI code locations page 
+    <ul>
+      <li>
+        <strong>Format</strong> - <code>string</code>
+      </li>
+    </ul>
+  </ReferenceTableItem>
+      <ReferenceTableItem propertyName="code_source.package_name">
+    Use if your dagster code is structured as a package
```

Comment:
```suggestion
    The name of a package containing Dagster code
```

Suggesting a rewrite here, which makes it consistent with the way other descriptions are worded.